frontend/src/components/ExternalLink.tsx

```diff
@@ -14,7 +14,7 @@ const ExternalLink: React.FC<ExternalLinkProps> = ({ text, to }) => (
     isInline
     onClick={() => {
       window.open(to);
-      fireTrackingEventRaw('ExternalLink Clicked', { href: to, from: window.location.pathname });
+      fireLinkTrackingEvent('ExternalLink Clicked', { href: to, from: window.location.pathname });
```

Comment:
Question, not request... did we want to create an Enum for all of these `eventName` or `eventType` like system? That way we can look at it in one spot and know what is already in place.